checks/check_extra7167

```diff
@@ -27,7 +27,7 @@ CHECK_CAF_EPIC_extra7167='Infrastructure security'
 extra7167() {
   if [[ "$($AWSCLI $PROFILE_OPT shield get-subscription-state --output text)" == "ACTIVE" ]]; then
     LIST_OF_CLOUDFRONT_DISTRIBUTIONS=$($AWSCLI cloudfront list-distributions $PROFILE_OPT --query 'DistributionList.Items[*].[Id,ARN]' --output text)
```

Comment:
have you test it when the variable is empty? if empty it will go thru the if and shouldn't. So better  to add at the end of line 29 something like: 
```suggestion
    LIST_OF_CLOUDFRONT_DISTRIBUTIONS=$($AWSCLI cloudfront list-distributions $PROFILE_OPT --query 'DistributionList.Items[*].[Id,ARN]' --output text | grep -v None)
```

Reply:
Have updated the code and tested on my side, works good, thankyouuu